### PR TITLE
contact page

### PR DESCRIPTION
**Pull request by liannarice**
fixed spacing in form, fixed background height and padding. just need to add in footer and header

### CODE REVIEW

File: css/contact.css [hunk 1]
```diff
@@ -35,16 +36,16 @@ textarea {
...
-	margin: 5px 0;
+	/* margin: 5px 0; */
 }
 .message {
 	min-height: 140px;
 	width: 100%;
 	border-radius: var(--spacing10);
 	border: none;
-	margin: var(--spacing10) 0;
+	/* margin: var(--spacing10) 0; */
```
Comment (SolivanLau) [inline css/contact.css]: Oh it looks like you left code that was commented out - please remove it before merging

File: css/contact.css [hunk 2]
```diff
@@ -35,16 +36,16 @@ textarea {
 	background-color: var(--color-white);
 	border: 1px solid #6c6c6c;
 	width: 100%;
-	border-radius: 5px;
+	border-radius: var(--spacing10);
 	padding: var(--spacing10);
-	margin: 5px 0;
+	/* margin: 5px 0; */
```
Comment (SolivanLau) [inline css/contact.css]: Some commented out code! please remove it when you get a chance to keep code clean

File: css/contact.css [hunk 3]
```diff
@@ -35,16 +36,16 @@ textarea {
 	background-color: var(--color-white);
 	border: 1px solid #6c6c6c;
```
Comment (SolivanLau) [inline css/contact.css]: is it possible to have border color a var? feel free to add it to **global.css**

**Review by SolivanLau — CHANGES_REQUESTED** [review]
Hello! visually the contact page looks good. 

A few changes are needed outside of mentioned comments:
1. feel free to remove the footer. we can update after the **footer PR** is merged in
2.  please add the open sans google font code (only the last `<link>` tag with the long url is necessary)
3.  feel free to remove boiler plate comments with instructions in **contact.html** and **contact.css** to keep things clean!

File: css/contact.css (hunk 1 above)
Comment (charlisable) [reply css/contact.css]: comment removed

File: css/contact.css (hunk 2 above)
Comment (charlisable) [reply css/contact.css]: comment removed

File: css/contact.css (hunk 3 above)
Comment (charlisable) [reply css/contact.css]: added variable to global.css as --color-grey and replaced in contact.css

**Comment by charlisable** [comment]
Addressed all comments, including removing the footer section for now.

**Review by SolivanLau — APPROVED** [review]
Approved! and ready to go, thank you!

File: css/contact.css (hunk 1 above)
Comment (SolivanLau) [reply css/contact.css]: marvelous

File: css/contact.css (hunk 2 above)
Comment (SolivanLau) [reply css/contact.css]: Astounding

File: css/contact.css (hunk 3 above)
Comment (SolivanLau) [reply css/contact.css]: Yay!

**Pull request merged**